city_scrapers/spiders/il_sex_offender_management.py

```diff
@@ -0,0 +1,249 @@
+import json
+import re
+from datetime import datetime, timedelta
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import BOARD, CANCELLED, PASSED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class IlSexOffenderManagementSpider(CityScrapersSpider):
+    name = "il_sex_offender_management"
+    agency = "Illinois Sex Offender Management Board"
+    timezone = "America/Chicago"
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+    start_urls = [
+        "https://www2.illinois.gov/idoc/" + "Pages/SexOffenderManagementBoard.aspx"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        for link in response.css(
+            "#ctl00_PlaceHolderMain_ctl01" + "__ControlWrapper_RichHtmlField a"
+        ):
+            link_text = " ".join(link.css("*::text").extract())
+            if "Meeting" not in link_text:
+                continue
+            if "Agenda" in link_text:
+                # scrape agenda pdf for meeting information
+                yield scrapy.Request(
+                    response.urljoin(link.attrib["href"]),
+                    callback=self._parse_documents,
+                    dont_filter=True,
+                )
+            # special case where link to video was uploaded rather than agenda
+            if "WebEx" in link_text:
```

Comment:
To be safe, we can make this and the other title conditionals case-insensitive with `.lower()`. It seems possible that someone could type in "Webex" instead of "WebEx" as some point